src/node/server/serverPluginVue.ts

```diff
@@ -498,20 +499,37 @@ async function compileSFCMain(
   return result
 }
 
+export type TemplateCompilerOptions = [TemplateCompiler, CompilerOptions]
```

Comment:
Here https://github.com/vuejs/vue-loader/pull/1645#pullrequestreview-393927977 Evan requested an option for a shorthand, which we also implemented. It may also be nice here (I also implemented this feature here: https://github.com/RobbinBaauw/vite/commit/9f76f26eafb1267f25485fac5e52b856627a789b)